index.html

```diff
@@ -885,6 +856,158 @@ <h5>Timing Properties</h5>
             </p>
           </section>
+
+          <section id="contentDescriptor">
+            <h5><code>&lt;content-descriptor&gt;</code> values</h5>
+            <p>The <a>Script Represents</a> and <a>Represents</a> properties
+              indicate which components of the <a>related media object</a>
+              the contents of the relevant object represent.
+            </p>
+            <p>The values permitted in those properties depend on the
+              <code>&lt;<dfn>content-descriptor</dfn>&gt;</code> syntactic definition
+              and its associated registry table.
+            </p>
+            <p><code>&lt;<a>content-descriptor</a>&gt;</code> has a value conforming to the following syntax:</p>
+            <div class="exampleInner">
+              <pre class="language-abnf nohighlight">
+    &lt;content-descriptor&gt;  # see registry table below
+    : &lt;descriptor-token&gt; ( &lt;descriptor-delimiter&gt; &lt;descriptor-token&gt; )*
+  
+    &lt;descriptor-token&gt;
+    : <a href="https://www.w3.org/TR/xmlschema-2/#NMTOKEN">xsd:NMtoken</a>
+
+    &lt;descriptor-delimiter&gt;
+    : ;  # SEMI-COLON U+003B
+              </pre>
+            </div>
+            <p><code>&lt;<a>content-descriptor</a>&gt;</code> has values that are delimiter separated ordered lists
+              of tokens.</p>
+            <p>A <code>&lt;<a>content-descriptor</a>&gt;</code> value <em>A</em> is a <dfn data-abbr="sub-type">content descriptor sub-type</dfn>
+              of another <code>&lt;content-descriptor&gt;</code> value <em>B</em> if B's ordered list of tokens is
+              present at the beginning of A's ordered list of tokens.
+            </p>
+            <aside class="example">
+              <table class="data">
+                <caption>Table demonstrating example values of <code>&lt;<a>content-descriptor</a>&gt;</code>
+                  and whether each is a  <a>sub-type</a> of the other.
+                </caption>
+                <thead>
+                  <tr>
+                    <th><code>&lt;content-descriptor&gt;</code> A</th>
+                    <th><code>&lt;content-descriptor&gt;</code> B</th>
+                    <th>Is B a <a>sub-type</a> of A?</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  <tr>
+                    <td><code>visual;text</code></td>
+                    <td><code>visual</code></td>
+                    <td>No</td>
+                  </tr>
+                  <tr>
+                    <td><code>visual;text</code></td>
+                    <td><code>visual;text</code></td>
+                    <td>Yes</td>
+                  </tr>
+                  <tr>
+                    <td><code>visual;text</code></td>
+                    <td><code>visual;text;location</code></td>
+                    <td>yes</td>
+                  </tr>
+                </tbody>
+              </table>
```

Comment:
This feels too complicated to me. Can we simply say:
Some descriptor values may be a prefix to other values, when these other values are more precise. For example, the value "visual" would be used when a script event represents anything the video part of the programme, while "visual;text" would be used only when the script event represents only text content appearing in the video part of the programme.

Reply:
I think we need to keep the token list approach, to avoid accidental matching of partial words. We also need an implementable normative definition that's unambiguous.

We can certainly add that informative wording though.